includes/Rest/Site.php

```diff
@@ -89,15 +89,18 @@ public function delete_site() {
     public function delete_option_by_prefix( $prefix ) {
         global $wpdb;
 
-        $wemail_options = $wpdb->get_results( "SELECT option_name FROM $wpdb->options WHERE option_name LIKE $prefix . '%'" );
-        foreach( $wemail_options as $option ) {
+        $wemail_options = $wpdb->get_results( $wpdb->prepare( "SELECT option_name FROM $wpdb->options WHERE option_name LIKE %s", '$prefix%' ) );
+
+        foreach ( $wemail_options as $option ) {
             delete_option( $option->option_name );
         }
     }
 
     public function delete_user_metadata( $keys ) {
+        global $wpdb;
+
         foreach ( $keys as $key ) {
-            delete_user_meta(0,$key);
+            $wpdb->query( $wpdb->prepare( 'DELETE  FROM wp_usermeta WHERE meta_key = %s', $key ) );
```

Comment:
- Here `wp_usermeta` table can be dynamic table prefix. It could be `wp_` or `anything`. I think `$wpdb` object has a usermeta property which store usermeta table name with proper prefix.
- Also, you can use `Where In` clause here to delete by running just one query rather then multiple where query.